app/src/main/java/com/esoxjem/movieguide/listing/MoviesListingFragment.java

```diff
@@ -56,7 +63,7 @@ public void onAttach(Context context) {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setHasOptionsMenu(true);
-        setRetainInstance(true);
+      //    setRetainInstance(true);
```

Comment:
Why was this commented out?

Reply:
The saveInstanceState bundle is be null  with this set to true, and we need the saved state after orientation change in the onViewCreated so that we can restore .Hence I removed it.